main.go

```diff
@@ -56,7 +57,7 @@ func main() {
 		fmt.Fprintf(os.Stderr, "cannot find binary %q in any of:\n", binary)
 		workspaces := filepath.SplitList(build.Default.GOPATH)
 		for i, workspace := range workspaces {
-			path := filepath.Join(workspace, "bin", binary)
+			path := filepath.Join(workspace, "bin", binaryName(binary))
```

Comment:
My first revision included the second instance change as well. However, from the command `binstale command` you get output like `cannot find binary command.exe in any of C:\GOPATH\command.exe`. I prefered `cannot find binary command in any of C:\GOPATH\command.exe` to match the provided arguments, but either works.